app/views/about/show.body.md

```diff
@@ -0,0 +1,19 @@
+### Empowering the developer
+
+I've been an independent developer for the past two years. And leads are rarely consistent. Sometimes I can barely keep up with the work and other times I struggle to find my next gig.
+
```

Comment:
I know you are the owner of the project, but at this point, this is a community project right, suggest you remove the wording "I". Just my opinion

Reply:
actually, never mind, this might not be a true community project yet like rails.